### PR TITLE
feat: tests for remaining create commands

### PR DESCRIPTION
**Pull request by CristiCanizales**
Implement tests for remaining create commands
[W-14866435](https://gus.lightning.force.com/a07EE00001iera1YAA)

Test runs: 
- https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/9066845894 ✅ 
- https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/9067695951 ✅

### CODE REVIEW

File: test/specs/templates.e2e.ts [hunk 1]
```diff
@@ -634,14 +639,18 @@ describe('Templates', async () => {
...
     const workbench = await (await browser.getWorkbench()).wait();
-    const sidebar = workbench.getSideBar();
-    const treeViewSection = await sidebar.getContent().getSection(projectName);
-    await treeViewSection.expand();
-    const lwcTestFolder = await treeViewSection.findItem('__tests__');
-    await lwcTestFolder?.select();
-    let testItem = await treeViewSection.findItem('lightningWebComponent1.test.js');
-    await testItem?.select;
-    await browser.keys([CMD_KEY, 'Delete']);
+    const pathToLwcTest = path.join(
```
Comment (peternhale) [inline test/specs/templates.e2e.ts:642]: @CristiCanizales why the change to use shell command instead of UI?

File: test/specs/templates.e2e.ts [hunk 2]
```diff
@@ -671,7 +680,12 @@ describe('Templates', async () => {
     expect(outputPanelText).not.toBeUndefined();
 
     // Check for expected item in the Explorer view.
-    testItem = await treeViewSection.findItem('lightningWebComponent1.test.js');
+    const sidebar = workbench.getSideBar();
+    const treeViewSection = await sidebar.getContent().getSection(projectName);
```
Comment (peternhale) [inline test/specs/templates.e2e.ts:694]: If this is a common pattern open sidebar for project, expand, find down the tree, then how can this be optimized for brevity?

**Review by peternhale — APPROVED** [review]

File: test/specs/templates.e2e.ts (hunk 1 above)
Comment (CristiCanizales) [reply test/specs/templates.e2e.ts:651]: UI wasn't working, and the deletion itself is not what we want to test but I need to delete the file so the command can successfully run, as when there's already a test file the command fails.

File: test/utilities/miscellaneous.ts [hunk 3]
```diff
@@ -103,3 +105,48 @@ export async function getTextEditor(workbench: Workbench, fileName: string): Pro
...
+  await inputBox.confirm();
+  await pause(1);
+
+  // Select the default directory (press Enter/Return).
+  await inputBox.confirm();
+
+  const successNotificationWasFound = await notificationIsPresentWithTimeout(
+    workbench,
+    `SFDX: Create ${type} successfully ran`,
+    TEN_MINUTES
```
Comment (peternhale) [inline test/utilities/miscellaneous.ts:130]: @CristiCanizales I know it is necessary to wait for a notification to appear, but why such a large wait time?

File: test/utilities/sideBar.ts [hunk 4]
```diff
@@ -8,14 +8,22 @@
 import { DefaultTreeItem, TreeItem, ViewItem, Workbench, ViewSection } from 'wdio-vscode-service';
 import { pause } from './miscellaneous';
 
+export async function expandSideBar(
```
Comment (peternhale) [inline test/utilities/sideBar.ts:11]: Thanks for creating a utility function.

File: test/testData/sampleAnalyticsTemplateData.ts [hunk 5]
```diff
@@ -0,0 +1,240 @@
+export const variables = [
+  '{',
```
Comment (peternhale) [inline test/testData/sampleAnalyticsTemplateData.ts:2]: @CristiCanizales why not use a plain object and then do a JSON.stringify?

**Review by peternhale — CHANGES_REQUESTED** [review]

File: test/utilities/miscellaneous.ts (hunk 3 above)
Comment (CristiCanizales) [reply test/utilities/miscellaneous.ts:130]: It's the default time we've been using for checking notifications across all the tests. I created [W-15749369](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001rODKCYA4/view) so we can work on improving all of this long times issues and hard pauses we have everywhere.

File: test/utilities/sideBar.ts [hunk 6]
```diff
@@ -19,6 +19,25 @@ export async function expandSideBar(
...
+  const visibleItemsLabels = await visibleItems.reduce(
+    async (previousPromise: Promise<string[]>, currentItem: DefaultTreeItem) => {
+      const results = await previousPromise;
+      const label = await currentItem.getLabel();
+      results.push(label);
+
+      return results;
+    },
+    Promise.resolve([])
+  );
```
[the remark this answers is not in the document]
Comment (CristiCanizales) [reply test/utilities/sideBar.ts]: Thanks! ✅

File: test/testData/sampleAnalyticsTemplateData.ts (hunk 5 above)
Comment (CristiCanizales) [reply test/testData/sampleAnalyticsTemplateData.ts:2]: I am following the exact same pattern we're using for the rest of the steps, for consistency

**Review by peternhale — APPROVED** [review]
